package.json

```diff
@@ -2,25 +2,17 @@
   "private": true,
   "repository": "https://github.com/thunderstore-io/thunderstore-ui",
   "workspaces": [
-    "nextjs",
```

Comment:
I don't think it really matters, keeps the configuration simpler to have everything under the same rules of structuring. If anything you could rename the `packages` to something like `projects`.

Reply:
I think `projects` sounds better since we're unlikely to have more libraries and the components are mostly for our own projects. I don't have a strong preference between the two though, it's pretty minor